libsolidity/formal/ModelCheckerSettings.cpp

```diff
@@ -37,12 +39,26 @@ map<string, TargetType> const ModelCheckerTargets::targetStrings{
 	{"outOfBounds", TargetType::OutOfBounds}
 };
 
+namespace
+{
+
+bool versionAtLeast080()
+{
+	return tuple(ETH_PROJECT_VERSION_MAJOR, ETH_PROJECT_VERSION_MINOR, ETH_PROJECT_VERSION_PATCH) >= tuple(0, 8, 0);
+}
```

Comment:
Can it ever return `false`? We're already well past 0.8.0 at this point.

Reply:
True